x-pack/plugins/security_solution/public/overview/links.ts

```diff
@@ -15,10 +15,16 @@ import {
 } from '../../common/constants';
 import { DASHBOARDS, DETECTION_RESPONSE, GETTING_STARTED, OVERVIEW } from '../app/translations';
 import { FEATURE, LinkItem } from '../common/links/types';
+import overviewPageImg from '../common/images/overview_page.png';
+import detectionResponsePageImg from '../common/images/detection_response_page.png';
 
 export const overviewLinks: LinkItem = {
   id: SecurityPageName.overview,
   title: OVERVIEW,
+  landingImage: overviewPageImg,
+  description: i18n.translate('xpack.securitySolution.appLinks.overviewDescription', {
+    defaultMessage: 'What is going in your secuity environment.',
```

Comment:
This might be `What is going on in your security environment`. Else the current sentence reads like something is being sent to the security environment.